CRM/Contribute/BAO/ManagePremiums.php

```diff
@@ -89,15 +83,14 @@ public static function setIsActive($id, $is_active) {
    * Add a premium product to the database, and return it.
    *
    * @param array $params
-   *   Reference array contains the values submitted by the form.
-   * @param array $ids
-   *   Reference array contains the id.
+   *   contains the values submitted by the form.
+   * @param int $productId
    *
    * @return CRM_Contribute_DAO_Product
    */
-  public static function add(&$params, &$ids) {
```

Comment:
This one feels like a change in the function's contract.

Reply:
Per my comment below - I think we HAVE standardised a lot of add & create functions - but we should try to remove $ids rather than change it as part of that - I think $params['id'] is the same as $productId

Reply:
Done